dot/sync/chain_sync.go

```diff
@@ -160,6 +161,13 @@ type chainSync struct {
 	slotDuration time.Duration
 }
 
+// ByPeerStateNumber implements sort.Interface to make the it sortable by peerState.number
+type ByPeerStateNumber []interface{}
```

Comment:
Forgive me if that's a dumb question, but can this me a slice of `*bit.Int` instead?

Reply:
I had to make the Outlier remover method generic by accepting `interface{}`. Hence, it needs to be in such a way.

Reply:
What else does it accept apart from `*big.Int`?

Reply:
In current version, `ByPeerStateNumber `'s removed altogether. I am currently using `*big.Int` slice.